lang/cs/Org.Apache.REEF.Common.Tests/Org.Apache.REEF.Common.Tests.csproj

```diff
@@ -61,8 +61,18 @@ under the License.
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
```

Comment:
If we move the change to build.prop, this ItemGroup should be moved to build.prop as well as every project requires it.